dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
+    }
+
+    // The Notification Device Data
+    [contract(AppNotificationsContract, 2)]
+    runtimeclass AppNotificationDevicesData
```

Comment:
What identity is used to open the video/audio feeds to show in the toast, and will be shown to the user as a "did you want to let <x> use your camera?" As it's your process, it's probably "Windows App SDK" and not "Contoso Calling".

Reply:
Camera Permission will be both Notification and App level which can be toggled by user from camera settings.
Since we are launching the camera process from Shell OS.